.github/workflows/ci.yml

```diff
@@ -20,6 +21,9 @@ jobs:
       with:
         node-version: ${{ matrix.node }}
 
+    - name: Install npm 8.5.x
```

Comment:
I have the same question here as we had in frontend-platform - I'd really expect this installation step isn't necessary.  You'd indicated tests break if this isn't here?  I'd love to see what's breaking and see if we can find a root cause to address.

Reply:
It seemed unnecessary, so I have removed it. Build is successful without it.